composer.json

```
@@ -19,7 +19,7 @@
   "require-dev": {
     "guzzlehttp/promises": "0.1.1|^1.3",
     "squizlabs/php_codesniffer": "^3.5",
-    "phpunit/phpunit": "^7.5||^8.5",
+    "phpunit/phpunit": "^9.0.0",
```

Let's see if this fixes the tests..
```suggestion
    "phpunit/phpunit": "^7.5||^9.0.0",
```